crates/symbolicator-service/src/services/symbolication/js.rs

```diff
@@ -213,23 +218,31 @@ async fn symbolicate_js_frame(
                 None => &Err(CacheError::NotFound),
             };
 
-            apply_source_context_from_artifact(&mut frame, source_file);
+            apply_source_context_from_artifact(&mut frame, source_file).ok();
```

Comment:
```suggestion
            apply_source_context_from_artifact(&mut frame, source_file)?;
```

Here and above, I believe we can just directly propagate these.
I don’t think it matters much as you are setting `frame.lineno` above, so this should be unreachable.